pkg/plugins/k8s/k8s_plugin.go

```diff
@@ -63,13 +73,15 @@ func (u *k8sUpdateTarget) String() string {
 }
 
 const (
-	switchdevManifestPath       = "bindata/manifests/switchdev-config/"
-	switchdevUnits              = switchdevManifestPath + "switchdev-units/"
-	switchdevUnitFile           = switchdevUnits + "switchdev-configuration.yaml"
-	networkManagerUnitFile      = switchdevUnits + "NetworkManager.service.yaml"
-	ovsUnitFile                 = switchdevManifestPath + "ovs-units/ovs-vswitchd.service.yaml"
-	configuresSwitchdevScript   = switchdevManifestPath + "files/configure-switchdev.sh.yaml"
-	switchdevRenamingUdevScript = switchdevManifestPath + "files/switchdev-vf-link-name.sh.yaml"
+	switchdevManifestPath             = "bindata/manifests/switchdev-config/"
+	switchdevUnits                    = switchdevManifestPath + "switchdev-units/"
+	switchdevBeforeNMUnitFile         = switchdevUnits + "switchdev-configuration-before-NM.yaml"
```

Comment:
I'm not sure that we want to have upper-case letters in file name but I won't block patch for it

Reply:
dont think its critical, although we dont use capitals for vf nor ovs.
@zshi-redhat WDYT shall we stick with lower case letters ?

Reply:
sure, updated to lowercase.

Reply:
thank you, @zshi-redhat